### PR TITLE
ci: Fixate CentOS container image until rpm-md repos sync

### PR DESCRIPTION
**Pull request by cgwalters**
Copy of https://github.com/projectatomic/rpm-ostree/pull/985

### CODE REVIEW

**Comment by jlebon** [comment]
@rh-atomic-bot r+ 682033e

**Comment by jlebon** [comment]
Added a small fixup; the comments in between the list entry lines were throwing off PyYAML. (Will need to tweak the parser to send a better update for this).

@rh-atomic-bot r+ 0ed28e0

File: .papr.yml [hunk 1]
```diff
@@ -41,9 +41,9 @@ env:
     CFLAGS: ''
 
 tests:
-  - docker run --privileged -v $PWD:$PWD --workdir $PWD
-    # FIXME revert this when repos sync
+    # FIXME revert setting to 7/3/1611 when repos sync
```
Comment (jlebon) [inline .papr.yml]: Whoops, I somehow typed `/` instead of `.` here!

**Comment by jlebon** [comment]
Hmm, there's some flakes in the admin tests.
@rh-atomic-bot retry

**Comment by cgwalters** [comment]
Split out a race bug to https://github.com/ostreedev/ostree/issues/1162

**Comment by cgwalters** [comment]
@rh-atomic-bot retry

**Pull request closed without merging**